scripts/combine-crossreg.R

```diff
@@ -0,0 +1,237 @@
+## Script to build TRN-TRN table for manual proofing using publications table, title matching, and TRN(registry data) table
+
+##########################################################
+
+library(here)
+library(fs)
+library(lubridate)
+library(tidyverse)
+library(ctregistries)
+
+
+# Load registry data
+trn_registry_data <- read_rds("data/cross-registrations/trn-registry-data.rds")
+
+# Load title matching data
+title_matches <- read_rds("data/cross-registrations/title-matched-7.rds")
+
+# Load the publications table once it's ready
+publications <- read_rds("data/cross-registrations/trn-publications.rds")
+
+# Get trials, limited to those meeting intovalue criteria
+source(here::here("scripts", "functions", "filter-intovalue-criteria.R"))
+trials <- read_iv_trials()
+
+
+####################################################################################################################
+
+## First build matches table from registry data table
+
+trn_trn_longer <- trn_registry_data |>
+  filter(!is.na(trns_reg), # filter rows where trns_reg is empty
+         trns_reg != "") |>
+  mutate(trn_split = strsplit(trns_reg, ";")) |> # Split trns_reg by semicolon to get individual trial IDs
+  unnest_longer(trn_split) |> # unnest longer to allow pairwise comparisons
+  filter(trn_split != id) |> # filter out self-references
+  rename(trn1 = id,
+         trn2 = trn_split) |>
+  distinct(trn1, trn2, .keep_all = TRUE) |>
+  relocate(trn1, trn2, everything()) |>
```

Comment:
Re @delwen's https://github.com/maia-sh/intovalue-data/pull/44#discussion_r1682889821, yes this looks like relocate now.